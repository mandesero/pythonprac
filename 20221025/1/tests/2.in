print(*fib(6, 12))