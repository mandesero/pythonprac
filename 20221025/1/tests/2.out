13, 21, 34, 55, 89, 144, 233
